package.json

```diff
@@ -8,9 +8,9 @@
     "@testing-library/user-event": "^13.5.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-rating-stars-component": "^2.2.0",
```

Comment:
Perfect! That error was annoying, thanks for finding a more up to date package +1